hugegraph-common/src/main/java/com/baidu/hugegraph/config/HugeConfig.java

```diff
@@ -59,35 +60,7 @@ public HugeConfig(String configFile) {
     }
 
     private void reloadIfNeed(Configuration conf) {
-        if (!(conf instanceof AbstractFileConfiguration)) {
-            if (conf instanceof AbstractConfiguration) {
-                AbstractConfiguration config = (AbstractConfiguration) conf;
-                config.setDelimiterParsingDisabled(true);
-            }
-            return;
-        }
-        AbstractFileConfiguration fileConfig = (AbstractFileConfiguration) conf;
-
-        File file = fileConfig.getFile();
-        if (file != null) {
-            // May need to use the original file
-            this.setFile(file);
-        }
-
-        if (!fileConfig.isDelimiterParsingDisabled()) {
-            /*
-             * PropertiesConfiguration will parse the containing comma
-             * config options into list directly, but we want to do
-             * this work by ourselves, so reload it and parse into `String`
-             */
-            fileConfig.setDelimiterParsingDisabled(true);
-            try {
-                fileConfig.refresh();
-            } catch (ConfigurationException e) {
-                throw new ConfigException("Unable to load config file: %s",
-                                          e, file);
-            }
-        }
+        //TODO impl reload
```

Comment:
need to implement?

Reply:
The reload implement is very complex for config2, could we not implement it?

Reply:
ok please check the following comment, or improve comment if need to keep it: `// TODO implement reloadIfNeed()`.

            /*
             * PropertiesConfiguration will parse the containing comma
             * config options into list directly, but we want to do
             * this work by ourselves, so reload it and parse into `String`
             */

Reply:
I have removed it